From: Frederic Peters <fpeters@debian.org>
Bug-Debian: http://bugs.debian.org/618764
Description: Do not consider system users returned from consolekit history.
Forwarded: not-needed
---
 src/daemon.c |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

--- accountsservice.orig/src/daemon.c
+++ accountsservice/src/daemon.c
@@ -336,6 +336,7 @@ process_ck_history_line (Daemon      *da
         gchar *username;
         gulong frequency;
         User *user;
+        struct passwd *pw;
 
         frequency = 0;
         username = NULL;
@@ -344,7 +345,9 @@ process_ck_history_line (Daemon      *da
                 return;
         }
 
-        if (user_is_excluded (daemon, username, daemon->priv->minimal_uid)) {
+        pw = getpwnam (username);
+
+        if (user_is_excluded (daemon, username, pw->pw_uid)) {
                 g_debug ("excluding user '%s'", username);
                 g_free (username);
                 return;
